internal/operators/ocs/validations.go

```diff
@@ -68,20 +68,21 @@ func setOperatorStatus(cluster *models.Cluster, status string) error {
 }
 
 // ValidateOCSRequirements is used to validate min requirements of OCS
-func (o *ocsValidator) ValidateOCSRequirements(cluster *models.Cluster) bool {
+func (o *ocsValidator) ValidateOCSRequirements(cluster *models.Cluster) string {
 	var status string
 	var err error
+	var validationPending bool
 	hosts := cluster.Hosts
 
 	if int64(len(hosts)) < o.OCSRequiredHosts {
 		status = "Insufficient hosts to deploy OCS. A minimum of 3 hosts is required to deploy OCS. "
 		err = setOperatorStatus(cluster, status)
 		if err != nil {
 			o.log.Error("Failed to set Operator status ", err)
-			return false
+			return "failure"
```

Comment:
it will be better to use const instead of "failure", "pending"

Reply:
fixed